cc/scripts/scripts.js

```diff
@@ -58,8 +69,8 @@ const miloLibs = setLibs(LIBS);
 }());
 
 (async function loadPage() {
+  await loadArticlePromotions();
   const { loadArea, loadDelayed, setConfig } = await import(`${miloLibs}/utils/utils.js`);
```

Comment:
```suggestion
  const { getMetadata, loadArea, loadDelayed, setConfig } = await import(`${miloLibs}/utils/utils.js`);
  await loadArticlePromotions(getMetadata);
```

Reply:
@auniverseaway I have integrated the mentioned changes from review comment.